javascript/homework04.js

```diff
@@ -0,0 +1,49 @@
+const commander = require("commander");
+const axios = require("axios");
+
+commander
+  .option("-c, --concurrency <n>", "Number of parallel requests", parseInt)
+  .option("-n, --requests <n>", "Number of requests to perform", parseInt)
+  .option("-b, --body", "Send a random generated body with request")
+  .arguments("<url>")
+  .action(async (url) => {
+    const start = Date.now();
```

Comment:
probably can be inside requests loop, otherwise my output looks strange

1 530
2 1039
3 1547
4 2054
5 2563
6 3070
7 3579
8 4087
9 4595
10 5103
11 5611
12 6118
13 6625
14 7132
15 7640